app/javascript/app/utils/map.js

```diff
@@ -0,0 +1,24 @@
+export const getDefaultStyles = () => ({
```

Comment:
The modularization is done inside the own component with [defaultProp](https://github.com/Vizzuality/climate-watch/blob/develop/app/javascript/app/components/map/map-component.jsx#L128) styles, but I thought that the default styles for the app (and not for the component) will be necessary for other map containers and that is why I've moved there.